frontend/src/components/common/ApiVersionInfo.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { IApiVersion, useApiHealth } from 'hooks/pims-api';
+
+/**
+ * Provides a way to display the API version information.
+ * Makes an AJAX request to the API for the version information.
+ * @returns ApiVersionInfo component.
+ */
+export const ApiVersionInfo = () => {
+  const api = useApiHealth();
+  const [version, setVersion] = React.useState<IApiVersion>();
+
+  React.useEffect(() => {
+    api.getVersion().then(response => {
```

Comment:
I think changing to `<div className="version">{`v${version?.version}`}</div>` would solve the loading issue.

Are you suggesting caching the api endpoint?  Technically there is little value in doing so.  Or do you mean using redux or context to store the value so that it's only called in one place?

Reply:
I don't think caching, redux or context needs to be used as long as we are sure this is the only place we will be using this component/the version api call.

Reply:
Less concerned about the above than I am about the missing error handling logic.